sentry_sdk/integrations/rust_tracing.py

```diff
@@ -205,48 +203,35 @@ def on_new_span(self, attrs: str, span_id: str) -> TraceState:
         else:
             sentry_span_name = "<unknown>"
 
-        kwargs = {
-            "op": "function",
-            "name": sentry_span_name,
-            "origin": self.origin,
-        }
-
-        scope = sentry_sdk.get_current_scope()
-        parent_sentry_span = scope.span
-        if parent_sentry_span:
-            sentry_span = parent_sentry_span.start_child(**kwargs)
-        else:
-            sentry_span = scope.start_span(**kwargs)
+        span = sentry_sdk.start_span(
```

Comment:
@matt-codecov @antonpirker I simplified this part since we don't want users to do scope stuff in integrations, it is anti-patterny

Reply:
from a quick look at the code, it looks like `__enter__()` will still update the current span and keep track of the parent so that `__exit__()` can restore it, right? if so then i am fine with this. not qualified to stamp, and also CI is unhappy

Reply:
yes exactly, span management should be done in the core and will just work
ci is unhappy because this is a dev branch